packages/fast-components-react-base/src/breadcrumb/breadcrumb.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure, mount, shallow } from "enzyme";
+import Breadcrumb, {
+    BreadcrumbClassNameContract,
+    BreadcrumbUnhandledProps,
+} from "./breadcrumb";
+import { KeyCodes } from "@microsoft/fast-web-utilities";
+
+/*
+ * Configure Enzyme
+ */
+configure({ adapter: new Adapter() });
+
+const managedClasses: BreadcrumbClassNameContract = {
+    breadcrumb: "breadcrumb-class",
+    breadcrumb_item: "breadcrumb-item-class",
+    breadcrumb_item__current: "breadcrumb-item-current-class",
+    breadcrumb_ol: "breadcrumb-ol-class",
+    breadcrumb_separator: "breadcrumb-separator-class",
+};
+
+describe("breadcrumb", (): void => {
+    test("should have a displayName that matches the component name", () => {
+        expect((Breadcrumb as any).name).toBe(Breadcrumb.displayName);
+    });
+
+    test("should not throw if managedClasses are not provided", () => {
+        expect(() => {
+            shallow(<Breadcrumb />);
+        }).not.toThrow();
+    });
+
+    test("should implement unhandledProps", (): void => {
+        const unhandledProps: BreadcrumbUnhandledProps = {
+            "aria-controls": "controls",
+        };
+
+        const rendered: any = shallow(<Breadcrumb {...unhandledProps} />);
+
+        expect(rendered.first().prop("aria-controls")).toEqual("controls");
+    });
+
+    test("should set current value on last rendered", (): void => {

```

Comment:
So, exists will ensure that a current value exists, but it can't ensure that it sets it on the last value. We'd need to drill into the children to do so.